js/letters.js

```diff
@@ -17,8 +17,24 @@ var counter = {
 
 function countLetters(counter, sample_text){
   // FIX ME
+  let i = 0;
+  let j = 0;
+  let next = 0 + i;
+  var count = 0;
+  let text = sample_text.toLowerCase();
+  if(text.length === 0){
+    return ' ';
+  }
+  let letter = text.charAt(next);
+  
+    if( [letter] ){
```

Comment:
In this particular case, you don't need the square brackets as this will always evaluate to true.

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.

Reply:
Also, formatting.